### PR TITLE
Tweak presentation of DPI info on the "About" form

### PR DESCRIPTION
**Pull request by drewnoakes**
Follows https://github.com/gitextensions/gitextensions/pull/5737#pullrequestreview-173691187

## Changes

- Tweak environment info output
 
### Before

image

```text
- Git Extensions 3.00.rc2
- Build 22c1cfd7e (Dirty)
- Git 2.19.1.windows.1
- Microsoft Windows NT 10.0.17134.0
- .NET Framework 4.7.3190.0
- DPI 144dpi (150% scaling)
```

### After

image

```text
- Git Extensions 3.00.rc2
- 5b18fe38e0c12c3ca8b85bb16d2f576
- Git 2.19.1.windows.1
- Microsoft Windows NT 10.0.17134.0
- .NET Framework 4.7.3190.0
- DPI 144dpi (150% scaling)
```

## Quality

- Manual testing

Has been tested on:
- GIT 2.19.1
- Windows 10

### CODE REVIEW

**Comment by drewnoakes** [comment]
@RussKie [suggested](https://github.com/gitextensions/gitextensions/pull/5757#issuecomment-438100738):

    Display 1920x1080, 96dpi, 100%
    Display 3840x2160, 192dpi, 200%

What's the use case here? We don't support per-monitor DPI.

File: GitUI/UserEnvironmentInformation.cs [hunk 1]
```diff
@@ -35,12 +35,13 @@ public static string GetInformation()
...
             StringBuilder sb = new StringBuilder();
 
-            sb.Append($"- Git Extensions {AppSettings.ProductVersion}{Environment.NewLine}");
-            sb.Append($"- {_sha} {(_dirty ? " (Dirty)" : "")}{Environment.NewLine}");
-            sb.Append($"- Git {gitVer}{Environment.NewLine}");
-            sb.Append($"- {Environment.OSVersion}{Environment.NewLine}");
-            sb.Append($"- {RuntimeInformation.FrameworkDescription}{Environment.NewLine}");
-            sb.Append($"- DPI X:{DpiUtil.ScaleX:P} Y:{DpiUtil.ScaleY:P}");
+            sb.AppendLine($"- Git Extensions {AppSettings.ProductVersion}");
+            sb.AppendLine($"- Build {_sha.Substring(0, 9)}{(_dirty ? " (Dirty)" : "")}");
```
Comment (gerhardol) [inline GitUI/UserEnvironmentInformation.cs]: The risk for ambiguity is low, but I still want o have the full sha or git to shorten the ref

Comment (vbjay) [reply GitUI/UserEnvironmentInformation.cs]: I don't want even a git shortened sha here.  I want the complete sha always.  I chose to do it that way on purpose.  You don't know if they are documenting something that needs to stand the passage of time.  Do not shorten at all this sha.  Github will shorten on display which is fine.  

image

**Comment by drewnoakes** [comment]
Updated, per feedback. The form is too short now. I'll try and get this updated tonight.

**Comment by vbjay** [comment]
LGTM.  

- Git Extensions 3.00.rc2
- Build 5b1713e4aa84b362d175d1eb28c46b738b503820
- Git 2.19.1.windows.1
- Microsoft Windows NT 10.0.17763.0
- .NET Framework 4.7.3190.0
- DPI 96dpi (no scaling)
- Primary screen resolution 2560 x 1080

image

**Comment by RussKie** [comment]
Throw a spanner in works - should we use textbox, this way it can scroll if
we cram more info

**Comment by RussKie** [comment]
> It might just be broken with UI scaling.

I'll check

**Comment by RussKie** [comment]
I think this is a sensible proposal.

The UI to have:
```
GitExtensions {version} {portable}
Build {sha}
Git {version}
```

The clipboard to have:
```
GitExtensions {version} {portable}
Build {sha}
Git {version}
.NET {version}
OS {version}
Monitor1 {resolution, dpi}
Monitor2 {resolution, dpi}
...
MonitorN {resolution, dpi}
```

Thoughts?

**Comment by vbjay** [comment]
Sure.  We could add a format specifier for the information class method and have it return one of the results.

**Comment by RussKie** [comment]
Merging as is. Let's continue the work though.

**Pull request merged**